MonoGame.Framework/Graphics/GraphicsDevice.DirectX.cs

```diff
@@ -719,7 +719,7 @@ internal void CreateSizeDependentResources(bool useFullscreenParameter = false)
                 using (var dxgiFactory = dxgiAdapter.GetParent<SharpDX.DXGI.Factory1>())
                 {
                     _swapChain = new SwapChain(dxgiFactory, dxgiDevice, desc);
-
+                    dxgiFactory.MakeWindowAssociation(PresentationParameters.DeviceWindowHandle, WindowAssociationFlags.IgnoreAll);
```

Comment:
So this disables `Alt+Enter` only it seems from the docs:

 https://msdn.microsoft.com/en-us/library/windows/desktop/bb174540(v=vs.85).aspx

Which I agree... we don't want the DX system controlling this, we want the GameWindow managing this itself.

@KonajuGames  @tgjones ???